src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -67,11 +67,6 @@ public BaseRecalibrationEngine( final RecalibrationArgumentCollection recalArgs,
 
         covariates = new StandardCovariateList(recalArgs, readsHeader);
 
-        logger.info("The covariates being used here: ");
-        for (final Covariate cov : covariates) { // list all the covariates being used
-            logger.info('\t' + cov.getClass().getSimpleName());
-        }
-
```

Comment:
Sure, but it gets printed once per 5k base pairs in my case, which spams the log. I've moved this to a utility function called by BaseRecalibratorSparkFn, so you still get the info but the "optimized" version doesn't print it quite so often.